test/integration/price-import.spec.coffee

```diff
@@ -87,16 +87,24 @@ describe 'Price Importer integration tests', ->
     prices = [
       {
         sku: 'sku1'
-        value:
-          centAmount: 9999
-          currencyCode: 'EUR'
+        prices: [
+          {
+            value:
+              centAmount: 9999
+              currencyCode: 'EUR'
+          }
+        ]
```

Comment:
in the price transformer, I do not have the product id. I have only sku, so how do I group by product? 
iNumber alone is not the product id.